python/py-genshi/Portfile

```diff
@@ -22,19 +23,12 @@ long_description \
    for parsing, generating, and processing HTML, XML or other textual \
    content for output generation on the web.
 
-homepage            http://genshi.edgewall.org/
-master_sites        http://ftp.edgewall.com/pub/genshi/
+homepage            https://genshi.edgewall.org
 
-distname            Genshi-${version}
-
-checksums           rmd160  3c8d8dfd8cb55b30e38af0039e2c42d56669c913 \
-                    sha256  1d154402e68bc444a55bcac101f96cb4e59373100cc7a2da07fbf3e5cc5d7352 \
-                    size    491579
+checksums           rmd160  61f88c9ac431d974aeeb5aa450b0a0868b3aa546 \
+                    sha256  52a13b8acf7aa96de5aae32bf6a28d7937c1540966e1100d2b9309c1e3b52af2 \
+                    size    279289
 
 if {${name} ne ${subport}} {
     livecheck.type      none
-} else {
-    livecheck.type      regex
-    livecheck.url       ${master_sites}
-    livecheck.regex     {Genshi-(\d+(?:\.\d+)*)\.[tz]}
 }
```

Comment:
the `livecheck.type none` is set by the `python` PG so the whole block can be removed.